packages/server/server/src/server/devServer.ts

```diff
@@ -121,6 +127,23 @@ export class ModernDevServer extends ModernServer {
 
     await super.onInit(runner, app);
 
+    if (this.useWorkerSSR) {
+      const { distDir, staticGenerate, conf, metaName } = this;
+      const ssrConfig = this.conf.server?.ssr;
+      const forceCSR =
+        typeof ssrConfig === 'object' ? ssrConfig.forceCSR : false;
+
+      // if we use worker ssr, we need override the routeRenderHandler
+      this.routeRenderHandler = createRenderHandler({
```

Comment:
It feels like a hidden logic here. Overwrites the routeHandler generated by prod-server. Is it better to do like this ?
```typescript
// prod-server
protected createRouterHandler() {
  this. routeRenderHandler = createRenderHandler()
}

// dev-server
protected createRouterHandler() {
  if (workerSSR) {
    this. routeRenderHandler =  createRenderHandler(workerSSRRender)
  } else {
    super()
  }
}
```

Reply:
done